### DISCUSSION TITLE
Welcome to DSharpPlus Discussions!

### DISCUSSION THREAD

**Discussion by Emzi0767** [DSharpPlus/DSharpPlus#716] [General]
# Brand-spanking-new discussions
Because some of you just seem allergic to Discord, even though you're using a library for it, we now have a discussions section on our GH so you can ask your questions here. Use this place for:
- discussion about subjects pertaining to the library
- questions
- other items that do not require tracking

This is not a place for:
- bug reports
- feature requests (but feature discussion is ok)
- etc

The overwatch would like to remind the citizens about the following links:
- [Our official Discord](https://discord.gg/KeAS3pU)
- [The docs](https://dsharpplus.github.io/)
- https://ko-fi.com/naamloos
- https://ko-fi.com/emzi0767
- https://paypal.me/Emzi0767/5USD
- https://patreon.com/emzi0767

**Comment by Naamloos** [comment]
<img src="https://cdn.discordapp.com/emojis/784477585139695626.png?v=1" width="50" height="50" alt="yep"/>

**Reply by uwx** [reply]
<img src="https://cdn.discordapp.com/emojis/784477585139695626.png?v=1" width="50" height="50" alt="yep"/>

**Reply by Giggitybyte** [reply]
![image](https://user-images.githubusercontent.com/19187704/102689057-240fd900-41b0-11eb-8884-1a48e3c95be6.png)

**Reply by ItsNovaAndCrap** [reply]
![102689057-240fd900-41b0-11eb-8884-1a48e3c95be6](https://user-images.githubusercontent.com/69518097/116237689-ec550700-a736-11eb-883d-bb364c7e07d0.png)

**Reply by ginomessmer** [reply]
![116237689-ec550700-a736-11eb-883d-bb364c7e07d0](https://user-images.githubusercontent.com/8465892/118370600-9f7f8600-b5a8-11eb-9b09-886cd3e959cb.png)

**Comment by Naamloos** [comment]
I'm just testing discussion webhooks.
